scripts/seeding_management.py

```
@@ -38,22 +59,30 @@ def find_tracker_config(client, torrent, config):
+        torrents: Optional list of torrents (avoids redundant API call if provided)
     """
-    try:
-        torrents = client.torrents.info()
-    except Exception:
-        logging.exception(f"Failed to fetch torrent list")
-        return
+    if torrents is None:
+        try:
+            torrents = client.torrents.info()
+        except Exception:
+            logging.exception(f"Failed to fetch torrent list")
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)